src/components/backgroundselector/style/backgroundselector.less

```diff
@@ -18,6 +18,36 @@
     }
   }
 
+  .ga-open-edit-bt {
+    position: absolute;
+    bottom: 2px;
+    left: 2px;
+    background-color: white;
```

Comment:
Are sure about this white background? kind of ugly no? I would at least use transparency and make it look like the thirdparty (same height and background)

Reply:
You don't see the icon otherwise. We can change that in the future if necessary, not a big deal.